Scoop/Controller/Ride Posting/InitialPostRideViewController.swift

```diff
@@ -0,0 +1,294 @@
+        }))
+        
+        setupTitleLabel()
+        setupStudentDriverButton()
+        setupTaxiButton()
+        setupDepartureTextField()
+        setupArrivalTextField()
+        setupLabels()
+    }
+    
+    private func setupTitleLabel () {
+        titleLabel.font = UIFont(name: "Rambla-Regular", size: 16)
+        titleLabel.text = "TRANSPORTATION METHOD"
+        titleLabel.accessibilityLabel = "preferred contact method"
+        titleLabel.textColor = .black
+        view.addSubview(titleLabel)
+        
+        titleLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(32)
+            make.top.equalToSuperview().inset(237)
+            make.height.equalTo(20)
+            make.trailing.equalToSuperview()
+        }
+    }
+    
+    private func setupStudentDriverButton() {
+        studentDriverButton.isSelected = true
+        studentDriverButton.setTitle("Student driver", for: .normal)
+        studentDriverButton.setTitleColor(.black, for: .normal)
+        studentDriverButton.titleLabel?.font = UIFont(name: "SFPro", size: 16)
+        studentDriverButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        studentDriverButton.setImage(UIImage(systemName: "circle"), for: .normal)
+        studentDriverButton.setImage(UIImage(systemName: "circle.inset.filled"), for: .selected)
+        studentDriverButton.tintColor = .black
+        studentDriverButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 0)
+        studentDriverButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 10)
+        studentDriverButton.addTarget(self, action: #selector(studentDriverButtonToggle), for: .touchUpInside)
+        view.addSubview(studentDriverButton)
+        
+        studentDriverButton.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).inset(-20)
+            make.leading.equalTo(titleLabel.snp.leading)
+            make.width.equalTo(180)
+        }
+    }
+    
+    private func setupTaxiButton() {
+        taxiButton.isSelected = false
+        taxiButton.setTitle("Taxi                  ", for: .normal)
+        taxiButton.setTitleColor(.black, for: .normal)
+        taxiButton.titleLabel?.font = UIFont(name: "SFPro", size: 16)
+        taxiButton.titleLabel?.adjustsFontSizeToFitWidth = true
+        taxiButton.setImage(UIImage(systemName: "circle"), for: .normal)
+        taxiButton.setImage(UIImage(systemName: "circle.inset.filled"), for: .selected)
+        taxiButton.tintColor = .black
+        taxiButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 0)
+        taxiButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 10)
+        taxiButton.addTarget(self, action: #selector(taxiButtonToggle), for: .touchUpInside)
+        view.addSubview(taxiButton)
+        
+        taxiButton.snp.makeConstraints { make in
+            make.top.equalTo(studentDriverButton.snp.bottom).inset(-20)
```

Comment:
use offset here again, using negative insets can usually be achieved using positive offsets which are more readble